src/rad/appdata.cpp

```diff
@@ -2367,6 +2367,7 @@ void ApplicationData::GenerateCode( bool panelOnly, bool noDelayed )
 	NotifyCodeGeneration( panelOnly, !noDelayed );
 }
 
+// Properties in ../output/xml/default.xml and code templates in ../output/xml/default.cppcode
```

Comment:
This comment is a bit misleading. This is generic code and applies to all languages, not only c++. Why is that path relative to `src`?

Reply:
My bad, I had that in there just to make it easier to switch between the three files in the editor. No reason for it to be in the PR.